proto/stride/interchainquery/v1/genesis.proto

```diff
@@ -15,7 +16,9 @@ message Query {
   string callback_module = 13;
   string callback_id = 8;
```

Comment:
When we decide to retry the query do we have to change the callback id?  Isn't the normal flow the icq handler follows that it catches the response, looksup the callback and data we prepared before we sent the query, executes whatever function is attached to the listener given the marshalled callback data, then it removes the callback data... so now we want to retry and we want the same callback data to still be there in case the icq works the second time.  Are we sure the callback won't have been removed on the first handling?

Reply:
man i see how this is confusing 😞 

ICQ callbacks are handled differently than ICA callbacks. For ICA callbacks you're right, but for ICQ callbacks, the callback data is just stored on the query object and the query response submission passes a query ID that does the lookup